### DISCUSSION TITLE
Is it possible to deploy this behind a reverse proxy?

### DISCUSSION THREAD

**Discussion by m3lv1lle** [Yooooomi/your_spotify#412] [Q&A]
It would be nice to be able to access it from everywhere.

**Comment by Jurrer** [comment]
Yes, you deploy it as any other application behind the proxy. 
You have to create two endpoints, web and backend. 
Be sure to specify them in the env vars later - this app must know what they are to generate shareable links and stuff

**Comment by webysther** [comment]
The LSIO bundle make this more easy: https://docs.linuxserver.io/images/docker-your_spotify/
